fe/fe-core/src/main/java/org/apache/doris/nereids/rules/RuleSet.java

```diff
@@ -243,6 +239,11 @@ public class RuleSet {
             .add(MaterializedViewFilterProjectAggregateRule.INSTANCE)
             .build();
 
+    public static final List<Rule> DPHYP_REORDER_RULES = ImmutableList.<Rule>builder()
+            .addAll(MATERIALIZED_VIEW_RULES)
```

Comment:
this maybe optmized later, MATERIALIZED_VIEW_RULES are also added by outside at `org.apache.doris.nereids.jobs.cascades.OptimizeGroupExpressionJob#execute`